.github/ISSUE_TEMPLATE/failing-test.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Failing Test
+about: File to report flaky and/or failing tests
+title: "[Failing Test] - [Test path here]"
+labels: failed-test, high, triage_needed
+assignees: ''
+
+---
+
+### Failing test name/path
+
+Add the failing test name here. Please don't create one issue per ci group with a failing test. Many times the first failing test will cause subsequent failing tests. Having separate issues for each one creates a lot of noise and adds overhead to track each one down.  We really only care about the first failed test.  If we fix that and a subsequent one continues to fail, we'll focus on that next.
+
+** A link to the failing build is helpful but these will expire so the rest of the information is important! **
```

Comment:
What if we add also the PR/commit that triggered that build? Is not always the root cause but can be helpful for newly flaky tests